test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,71 @@
+require "test_helper"
+
+class CommentsControllerTest < ActionDispatch::IntegrationTest
+  test 'should create a comment' do
```

Comment:
Some of your specs are divided in `preparation`, `action` and `expectations` that's cools!
But some others are not :/

Reply:
:o I didn't notice that. could you tell me please which test I could improve? I'd be great :D